packages/amazonq/src/extensionShared.ts

```diff
@@ -28,15 +28,14 @@ import {
     getLogger,
     getMachineId,
 } from 'aws-core-vscode/shared'
-import { initializeAuth, CredentialsStore, LoginManager, AuthUtils } from 'aws-core-vscode/auth'
+import { initializeAuth, CredentialsStore, LoginManager, AuthUtils, CommonAuthWebview } from 'aws-core-vscode/auth'
 import { makeEndpointsProvider, registerCommands } from 'aws-core-vscode'
 import { activate as activateCWChat } from 'aws-core-vscode/amazonq'
 import { activate as activateQGumby } from 'aws-core-vscode/amazonqGumby'
 import { CommonAuthViewProvider } from 'aws-core-vscode/login'
```

Comment:
there is a login export already, we should probably move it to /auth or move CommonAuthWebview to /login

Reply:
Good call, updated